openmp/libomptarget/DeviceRTL/CMakeLists.txt

```diff
@@ -301,7 +301,10 @@ endforeach()
 
 # Archive all the object files generated above into a static library
 add_library(omptarget.devicertl STATIC)
-set_target_properties(omptarget.devicertl PROPERTIES LINKER_LANGUAGE CXX)
+set_target_properties(omptarget.devicertl PROPERTIES
+                      LINKER_LANGUAGE CXX
+                      ARCHIVE_OUTPUT_DIRECTORY ${LLVM_LIBRARY_OUTPUT_INTDIR}
```

Comment:
Will this break standalone build?

Reply:
I was thinking about that and wasn't sure. As far as I can tell, `add_llvm_library` uses this string internally, which is how `libomptarget` ends up where it does. So I'm thinking so long as we're using that, it will be defined somewhere. But I haven't tested it or anything.

Reply:
I guess that's because `install(TARGETS omptarget.devicertl ARCHIVE DESTINATION ${OPENMP_INSTALL_LIBDIR})` overrides it, which is what we need; otherwise it will be installed into where LLVM is installed, which is not right if build standalone.

Reply:
I could change it to only do this if it's standalone, should probably be fine.

Reply:
Can we build the DeviceRTL standalone? Do we want to support that?

Reply:
I guess theoretically? I've never heard of anyone doing it though. It should ideally just require an up-to-date `clang` compiler and an LLVM installation.

Reply:
I'm constantly using standalone build.

Reply:
Can you test this then? Not really sure what the expected output would be. I guess the issue happens with tests, but do we even support tests out of tree?